components/_patterns/01-atoms/03-lists/01-ol.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * Available variables:
+ * - ol_base_class - the base classname
+ * - modifiers - array of modifiers to add to the base classname
+ * - blockname - blockname prepended to the base classname
+ *
+ * - listItems - TBD
+ */
+#}
+{% set ol_base_class = ol_base_class|default('ol') %}
+
+<ol{{ bem(ol_base_class, (modifiers), blockname) }}>
+  {% for ol_item in ol_items %}
+    {% include "@atoms/03-lists/_list-item.twig"
+      with {
```

Comment:
I don't know if we've talked about this, but I think we should standardize on having the `with {` not break to a new line. So this section would become:
```
{% include "@atoms/03-lists/_list-item.twig" with {
  "list_item_label": ol_item.label,
  "list_item_content": ol_item.content,
} %}
```